src/main/java/ch/njol/skript/structures/StructEvent.java

```diff
@@ -15,7 +15,47 @@
 
 import java.util.Locale;
 
-@NoDoc
+@Name("Event")
+@Description("""
+	The structure used for listening to events.
+	
+	Optionally allows specifying whether to listen to events that have been cancelled or events that have
+	been cancelled, and then were uncancelled again.
```

Comment:
```suggestion
	Optionally allows specifying whether to listen to events that have been cancelled or events that have
	been cancelled, and then were uncancelled again.
```
Not sure what you mean by the second half of this sentence. An event that has been cancelled and then uncancelled is just an uncancelled event. Skript doesn't have any special handling for that.